src/data/zmod/basic.lean

```diff
@@ -126,15 +123,6 @@ def comm_ring (n : ℕ) : comm_ring (fin (n+1)) :=
 
 local attribute [instance] fin.add_comm_semigroup
```

Comment:
```suggestion
```
I think this no longer does anything since the namespace ends immediately afterwards. (Actually, it's not clear if it did anything before, since it's also on line 74 which is in the same namespace.)

Reply:
Removed it.